app/views/users/show.html.erb

```diff
@@ -0,0 +1,26 @@
+<div class="container-lg" style="margin-top: 20px">
+  <div class="row">
+    <div class="col">
+      <h2>
+        My Trips
+      </h2>
+    </div>
+    <div class="col" style="text-align: right">
+      <div class="row">
+        </br>
+      </div>
+      <div class="row">
+        <div class="col" style="text-align: right">
+          <a href="/new_trip" style="color: black">
+            Create trip
+            <i class="fas fa-plus-square"></i>
+          </a>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<% @user.own_trips.each do |trip| %>
+  <%= render "trips/trip", trip: trip %>
+<% end %>
```

Comment:
Since this partial is [conventionally named](https://api.rubyonrails.org/classes/ActionView/PartialRenderer.html) you can shorten this to 

```erb
<% @user.own_trips.each do |trip| %>
  <%= render trip %>
<% end %>
```

BUT you can do even better and render a collection of partials

```suggestion
<%= render @user.own_trips %>
```
What do you think?